prow/plugins.yaml

```diff
@@ -12,6 +12,7 @@ triggers:
   - istio/old_mixer_repo
   - istio/old_pilot_repo
   - istio/test-infra
+  - istio-releases/daily-release
   trusted_org: istio
```

Comment:
@sebastienvas Should I add istio-releases to the trusted_org here?

Reply:
trusted org, means we trust the user from this org. Since we are part of both orgs, this should not be required.